lib/method.js

```diff
@@ -69,3 +77,9 @@ method.restore = function restoreMocks() {
   requireMocks = [];
   */
 };
+
+method.isMocked = function isMocked(obj, key) {
+  var flag = cache.get(obj);
+  if (!flag) return false;
```

Comment:
This project has curly brackets around if statements, even one liners. Or you could also do `return flag && flag.has(key);`